models/reviews-models.js

```diff
@@ -0,0 +1,24 @@
+const pg = require("pg");
+const format = require("pg-format");
+const db = require("../db/connection.js");
+
+exports.selectReviews = () => {
+  return db
+    .query(
+      `
+      SELECT reviews.*, COUNT(comments.review_id) AS comment_count
+      FROM reviews
+      LEFT JOIN comments
+      ON reviews.review_id = comments.review_id
+      GROUP BY reviews.review_id
+      ORDER BY reviews.created_at DESC;
+      `
+    )
+    .then((result) => {
+      const reviewArr = result.rows;
+      reviewArr.forEach((reveiw) => {
```

Comment:
Small spelling mistake that doesn't break the code (as you've been consistent), not an issue, just update for easier reading :)